tests/unit/coordinates_test.bzl

```diff
@@ -114,6 +115,17 @@ def _multiple_formats_impl(ctx):
 
 multiple_formats_test = unittest.make(_multiple_formats_impl)
 
+def _unusual_version_format_impl(ctx):
+    env = unittest.begin(ctx)
+
+    unpacked = unpack_coordinates("group:artifact:FY21R16")
```

Comment:
Should we also add a test for a version that doesn't have any numbers?

Reply:
Done